src/styles/components/_landingLayout.scss

```diff
@@ -15,3 +15,34 @@
     width: 100%;
   }
 }
+
+.landing-sub-container {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  min-width: 400px;
+  text-align: center;
+}
+
+.landing-logo { margin-top: 70px; }
+
+.landing-header {
+  letter-spacing: 3px;
+  line-height: 27px;
+  margin-bottom: 13px;
+  margin-top: 16px;
+}
+
+.landing-subtitle {
+  line-height: 32px;
+  margin: 0 0 10px 0;
+}
+
+.landing-description {
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 26px;
+  margin: 0 28% 49px 28%;
```

Comment:
when `y` margins are different but `x` margins are the same you can do `margin: 0 28% 49px` (top sides bottom)